default-recommendations/mutability-predicates.rkt

```diff
@@ -0,0 +1,60 @@
+#lang racket/base
+
+
+(require racket/contract/base)
+
+
+(provide
+ (contract-out
+  [mutability-predicates refactoring-suite?]))
+
+
+(require rebellion/private/static-name
+         resyntax/base
+         syntax/parse)
```

Comment:
Needs the `racket/mutability` import. Also, there's no need to import `rebellion/private/static-name`.

```suggestion
(require racket/mutability
         resyntax/base
         syntax/parse)
```